### DISCUSSION TITLE
What is [8 digit number]+[username]@users.noreply.github.com

### DISCUSSION THREAD

**Discussion by Veerhan-glitch** [community/community#41101] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

As you can guess from the title i want to know what is <8 digit number>+<username>@users.noreply.github.com

**Comment by airtower-luna** [comment, marked as answer]
It's a pseudo email address that can be used as the author email in commits to have them associated with a GitHub account without revealing the account owner's actual email address. "Pseudo" means it's not an actual address that can receive mail, it just looks like one.

See also the [description in the GitHub documentation](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address#about-commit-email-addresses), it also says what to do if you want to use this feature.

**Comment by KikeE36** [comment]
USERNAME@users.noreply.github.com

**Comment by asdwastakenig** [comment]
cool

**Comment by 1613kosmos1613-cmyk** [comment]
> ### Select Topic Area
> 
> Question
> 
> ### Body
> 
> As you can guess from the title i want to know what is <8 digit number>+<username>@users.noreply.github.com

**Comment by bhadreshmelvina123-creator** [comment]
bhadreshmelvina123@gmail.com

**Reply by muhammetakdag001-png** [reply]
Hi
